<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@848.v5a_383b_d14921">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@3769.v8b_e595e4d40d">
    <parentIds>
      <string>48</string>
    </parentIds>
    <id>49</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@3769.v8b_e595e4d40d">
      <arguments>
        <entry>
          <string>script</string>
          <string>
                            git clone -c advice.detachedHead=false --depth 1 --branch 1.0.2 https://9da9-2406-7400-81-a9e1-10bf-56c4-6e5b-9e17.ngrok-free.app/shiva/New 145
                        </string>
        </entry>
      </arguments>
      <sensitiveVariables/>
      <isUnmodifiedBySanitization>true</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@1259.vb_47f14fffc8a_">
      <startTime>1708598940704</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <wf.a.ErrorAction plugin="workflow-api@1259.vb_47f14fffc8a_">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 128</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:668)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:614)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:555)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions>
          <wf.a.ErrorAction_-ErrorId>
            <stackTrace/>
            <suppressedExceptions class="empty-list"/>
            <uuid>e26ff071-2e84-495a-8f1a-480a7d30c397</uuid>
          </wf.a.ErrorAction_-ErrorId>
        </suppressedExceptions>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>